include/unicorn/platform.h

```diff
@@ -151,9 +151,17 @@ typedef _W64 unsigned int  uintptr_t;
 // sys/time.h compatibility
 #if defined(_MSC_VER)
 #include <sys/timeb.h>
-#include <winsock2.h>
 #include <windows.h>
 
+// timeval is defined in winsock.h and winsock2.h
+// FD_ZERO is defined in both of these just above struct timeval.
+#if !defined(FD_ZERO)
+struct timeval {
+    long tv_sec;
+    long tv_usec;
+};
+#endif
```

Comment:
Usually it is defined in sys/time.h which is a linux/posix only header.

Previously it was being defined inside the winsock header, for winsock posix compatibility, but since we are removing the winsock related code I just define that one struct.